app/routes/raffles/$raffleId/index.tsx

```diff
@@ -328,9 +332,11 @@ export default function Index() {
                       {formatDateTime(endDateTime)}{" "}
                     </p>
                   </div>
-                  <div className="mb-8 text-sm text-neutral-700">
-                    {getRaffleActivitySubtitle(raffleActivityStatus)}
-                  </div>
+                  {raffleActivityStatus === RaffleActivityStatus.UPCOMING ? (
```

Comment:
Do you want to show a label even when you have a status tag on the title?

Reply:
Thanks this is great!